.claude/scripts/lyra-vm-harness.sh

```
@@ -0,0 +1,258 @@
+
+    # Process sample — 5 seconds
+    # note: `sample` requires sudo to sample other users' processes on macOS
+    ssh_run "$ip" "pid=\"\$(pgrep -x lyra | head -1)\"; \
+        if [ -n \"\$pid\" ]; then sudo sample \"\$pid\" 5 -f /tmp/lyra-vm-sample.txt 2>/dev/null; \
+        else printf 'lyra not running\n' > /tmp/lyra-vm-sample.txt; fi" 2>/dev/null && \
+        scp_get "$ip" "/tmp/lyra-vm-sample.txt" "$out_dir/process-sample.txt" && \
+        log "  process sample -> $out_dir/process-sample.txt" || \
+        log "  WARNING: process sample failed"
+
+    # Daemon log written by run-lyra
+    scp_get "$ip" "~/.lyra-vm-daemon.log" "$out_dir/daemon.log" 2>/dev/null && \
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)